migrations/versions/2628a398bab8_.py

```diff
@@ -0,0 +1,28 @@
+"""
+
+Revision ID: 2628a398bab8
+Revises: 5a5373c0d0d0
+Create Date: 2018-08-23 00:44:05.495949
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '2628a398bab8'
+down_revision = '5a5373c0d0d0'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('volunteer', sa.Column('allow_comms_during_event', sa.Boolean(), nullable=False))
```

Comment:
<@ms7821> the only thing I spotted is you'll need to add a server_default=expression.false() to this https://github.com/emfcamp/Website/pull/756/files#diff-8960f7762c417815fab99c7b646ea18dR19
11:45:40

<@ms7821> check out migrations/versions/716a1e3b4f86_add_cfp_accepted_only.py for an example